quipucords/quipucords/settings.py

```diff
@@ -30,6 +30,10 @@
 
 PRODUCTION = bool(os.environ.get("PRODUCTION", False))
 
+DISABLE_THREADED_SCAN_MANAGER = bool(
+    os.environ.get("DISABLE_THREADED_SCAN_MANAGER", False)
```

Comment:
We have a non-written tradition of prefixing our env vars with `QPC_` (most of them are following this pattern). I'd change the env var to follow this pattern.

On the other hand, considering this variable will probably be removed in the near future, I won't mind if the prefix is missing.

Reply:
Good point. I'll add `QPC_` to this!